pom.xml

```diff
@@ -189,6 +193,33 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>org.pitest</groupId>
+          <artifactId>pitest-maven</artifactId>
+          <version>1.1.11</version>
+          <configuration>
+            <reportsDirectory>local/pit-reports</reportsDirectory>
+            <timestampedReports>false</timestampedReports>
+            <timeoutConstant>1000</timeoutConstant>
+            <targetClasses>
+              <param>${pitest.classes}</param>
```

Comment:
This allows to limit the analysis from the whole module to a selected package. The full analysis takes over 30 minutes on optaplanner-core so limiting to a package is handy. Unfortunately the plugin itself doesn't seem to support parameter tweaking through system properties.